.travis.yml

```
@@ -107,8 +104,13 @@ jobs:
       # Use Binder build API to trigger repo2docker to build image
       - bash binder/trigger_binder.sh https://mybinder.org/build/gh/diana-hep/pyhf/"${TRAVIS_BRANCH}"
     after_success: skip
-  - stage: test_deploy
+  - stage: deploy
+    # Tests deployment to PyPI for all commits to master
     name: "TestPyPI deploy"
+    if: (branch = master) AND (NOT (type IN (pull_request)))
```

Better
```suggestion
    if: branch = master AND type = push
```

Just because otherwise it'll be triggered on periodic cron builds and api (or manual via Travis UI) builds which will conflict in pypi